### PR TITLE
Update all icons to have default 19px height; add new icons; revise some existing icons; add docs for devs

### PR DESCRIPTION
**Pull request by Scotchester**
Previously, icons were very large in their native SVG format and had to be scaled down with CSS. Icons now have a more reasonable default size when viewed without SVG, and can be scaled up with CSS in the same way they were previously scaled down.

## Additions

- Adds a whole category of new icons for expenses
- Adds some new icons to existing categories
- Added TOC to the top of the "Icon library" section so people can jump to a specific category.

## Changes

- Revises all icons to fix sizing issue describe above
- Fixes the SVGO config file so that the `process-icon-svgs` script now works as desired.

## Testing

1. Take a look at https://deploy-preview-1234--cfpb-design-system.netlify.app/design-system/foundation/iconography and make sure everything is in order
2. For extra credit, test out the steps in the new docs in `CONTRIBUTING.md`

## Todos

- [x] Update guidance in `CONTRIBUTING.md` for adding new icons.

## Checklist

- [x] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [x] Passes all existing automated tests
- [ ] Placeholder code is flagged / future todos are captured in comments
- [x] Project documentation has been updated
- [x] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

**Comment by anselmbradford** [comment]
Should `pricetag` be `price-tag` to fit with other two word icon names we use, such as `external-link`, `folder-save`, `credit-card`, etc.

Also, this is great!!

**Review by anselmbradford — APPROVED** [review]

**Comment by Scotchester** [comment]
@anselmbradford Added some docs in the most recent commit. Would you mind checking those out and letting me know what you think?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -217,3 +217,36 @@ In most cases styles should be declared mobile first,
...
+   - It should be a no-op for icons that have not changed
+1. If the icon has aliases, go into the icons folder and duplicate the canonical file to the aliases:
+   ```shell
+   cd packages/cfpb-icons/src/icons/
+
+   # Edit the command below for your actual canonical filename and aliases
+   for ALIAS in {one,two,three,}; do cp canonical.svg $ALIAS.svg; done
+   # Be sure to leave in the trailing comma if there is only one alias,
+   # or it will not be treated as an iterable object and will include
+   # the curly braces int he filename.
```
Comment (anselmbradford) [inline CONTRIBUTING.md:239]: Typo: `int he`

**Comment by anselmbradford** [comment]
@Scotchester one small typo, but otherwise looks good.

**Review by BBloomCFPB — APPROVED** [review]
I don't see anything amiss from my end.

**Pull request merged**